ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/cco-manual-security-token-service/deprovision/cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-deprovision-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-deprovision
+  steps:
```

Comment:
Suggest to insert `gather-mirror-registry` step here.

Reply:
Done